source/utils.js

```diff
@@ -58,6 +58,11 @@ function fipsToPostal(fips) {
   return fipsHash[fips].postal
 }
 
+function isDevEnviornment() {
```

Comment:
Minor, but you could probably just set a variable at load time and return that instead of re-checking every time this is called…